templates/cluster-template-windows.yaml

```diff
@@ -273,14 +273,14 @@ apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: AzureMachineTemplate
 metadata:
   annotations:
-    runtime: dockershim
+    runtime: containerd
```

Comment:
Shouldn't this template be a rename of cluster-template-windows-containerd.yaml now? I wouldn't expect us to keep both side by side

Reply:
I agree, I wasn't sure if we needed to keep them because of https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2292#pullrequestreview-968335791